docs/ref/templates/builtins.txt

```diff
@@ -673,9 +673,18 @@ if it's missing or has syntax errors), the behavior varies depending on the
 :class:`template engine's <django.template.Engine>` ``debug`` option (if not
 set, this option defaults to the value of :setting:`DEBUG`). When debug mode is
 turned on, an exception like :exc:`~django.template.TemplateDoesNotExist` or
-:exc:`~django.template.TemplateSyntaxError` will be raised; otherwise
-``{% include %}`` silences any exception that happens while rendering the
-included template and returns an empty string.
+:exc:`~django.template.TemplateSyntaxError` will be raised. When debug mode
+is turned off, ``{% include %}`` logs any exception that happens while
+rendering the included template with a level of WARNING and returns an empty
+string.
+
+.. versionchanged:: 1.9
+
+    Exceptions raised during template inclusion are logged using the
```

Comment:
Try to make this note "self-contained" as described in [Documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features). It should be minimal, e.g. "The logging described in the previous paragraph was added."